package.json

```diff
@@ -46,8 +52,11 @@
       "mocha": true
     },
     "rules": {
-      "semi": ["error", "always"],
-      "space-before-function-paren": ["error", "never"]
+      "no-param-reassign": "off",
```

Comment:
used in processing ops, now that i have it all working, i could go back and try and remove this one.  I'd rather open another issue if you want to go that route than tie it in with this one....